### ISSUE TITLE
[Liteloader-1.7.10] Just a little second of your time...

### ISSUE THREAD

**Issue by leopoldhub** [Pokechu22/WorldDownloader#87]
Hello, i need help about the svn link for liteloader... i know its not your job to help for that but in a post, you say you have setting up this workspace so i ask it to you... in the tutorial, we need to connect to " https://subversion.assembla.com/svn/liteloader/ " but it doesen't exist... and you have made it so, can you explain quickly how you made for this? thanks ^^ (link for the tutorial: [http://www.liteloader.com/explore/docs/dev:tutorial:source](url))

**Comment by Pokechu22** [comment]
They have a [gitlab instance](http://develop.liteloader.com/liteloader/LiteLoader) now.  As [1.7.10 is over 4 years old](https://howoldisminecraft1710.today/) and isn't being maintained by the liteloader team anymore, there really isn't an obvious branch for it; I used commit `d97239f428f98f34f4d0eae2a8060f9be99b6b99` though.  I don't know exactly how to explain the process I used, unfortunately; it was a mess with a lot of steps that didn't work right and I can't explain exactly what I did that worked.  @uyjulian may be able to help you more than I can.
